pkgs/applications/editors/emacs-modes/melpa-stable-packages.nix

```diff
@@ -36,7 +36,7 @@ self:
 
     overrides = {
       # upstream issue: mismatched filename
-      ack-menu = markBroken super.ack-menu;
+      # ack-menu = markBroken super.ack-menu;
```

Comment:
Are those supposed to be commented out?

Reply:
Yes, they were omitted from the generated output.